pkg/route/api/v1/types.go

```diff
@@ -43,6 +43,10 @@ type RouteSpec struct {
 	// be defaulted to Service.
 	To kapi.ObjectReference `json:"to"`
 
+	// SecondaryServices is an extension of the 'to' field. If more than one service needs to be
+	// pointed to, then use this field
+	SecondaryServices []kapi.ObjectReference `json:"secondaryServices,omitempty"`
```

Comment:
I'm not sure on the naming yet, but I'm leaning to "alternates" or "additionalTo".  We are not assuming services as target forever.